### ISSUE TITLE
New sniff to check @param is used for parameters

### ISSUE THREAD

**Issue by adaamz** [slevomat/coding-standard#639]
eg. this violates cs:
```php
/**
 * @var stdClass|string $a
 */
function f($a)
{}
```

should be
```php
/**
 * @param stdClass|string $a
 */
function f($a)
{}
```

**Comment by bkdotcom** [comment]
Squiz.Commenting.FunctionComment

however
it requires  'integer' and 'boolean' (php considers classnames) vs the correct 'int' and 'bool'

it also refuses to recognize '{@inheritdoc}'

```php
/**
 * {@inheritDoc}
 */
```

**Comment by adaamz** [comment]
@kukulich Can you tell me which name or how to find it?

**Comment by kukulich** [comment]
I think it's `FunctionCommentSniff`
